### DISCUSSION TITLE
Unintuitive line movements with relative line numbers

### DISCUSSION THREAD

**Discussion by Keshyu** [helix-editor/helix#11042] [Ideas]
(This is an attempt to re-open #10778)

# Summary
Both `<n>j` & `<n>k` jump visual lines; however, relative line numbers mark textual lines. As a result, trying to jump several lines using relative line numbers results in unexpected behaviour when some lines are wrapped.

# Details
In the original issue it was said that this is expected behaviour as per [the FAQ](https://github.com/helix-editor/helix/wiki/FAQ#can-the-jk-bindings-be-changed-to-ignore-soft-wrapping-when-using-a-count-like-3j) but the FAQ only talks about why `j/k` move visually by default (which I do agree with) when the real issue is the way relative line numbers work.

Absolute line numbers being textual makes sense for following compiler errors and the like. Relative line numbers, however, are solely used for jumping multiple lines using `j/k` (at least from my experience) so making line numbers textual when `j/k` are visual by default doesn't seem to be reasonable.

# Proposition
Make relative line numbers mark visual lines of text by default or at least make it configurable to do so as it is more intuitive in combination with `<n>j` & `<n>k`.

**Comment by valterschutz** [comment]
I also had this issue, but <n>g<j/k> was what I was looking for. I definitely agree that a config option that changes how relative line numbers are interpreted would be nice.
